cypress/integration/unit/logic/schedule-parser/shift-info.parser.spec.ts

```diff
@@ -5,24 +5,23 @@
 import { DataRowParser } from "../../../../../src/logic/schedule-parser/data-row.parser";
 import { MetaDataParser } from "../../../../../src/logic/schedule-parser/metadata.parser";
 import { ShiftsInfoParser } from "../../../../../src/logic/schedule-parser/shifts-info.parser";
+import { WorkerType } from "../../../../../src/common-models/worker-info.model";
 
 //#region  data declaration
 
-const TestCases: { data: DataRowParser[]; expectedData: DataRowParser[]; errorCount: number }[] = [
+const TestCases: { data: string[][]; expectedData: DataRowParser[]; errorCount: number }[] = [
   {
-    data: [
-      new DataRowParser(["opiekunka 1", "N", "Error", "N", " ", "D", "N", "NotValid", "L4", "U"]),
-    ],
+    data: [["opiekunka 1", "N", "Error", "N", " ", "D", "N", "NotValid", "L4", "U"]],
     expectedData: [
       new DataRowParser(["opiekunka 1", "N", "W", "N", "W", "D", "N", "W", "L4", "U"]),
     ],
     errorCount: 2,
   },
   {
     data: [
-      new DataRowParser(["opiekunka 1", "N", "Error", "N", " ", "D", "N", "NotValid", "L4", "U"]),
-      new DataRowParser(["opiekunka 2", "NotVald", "L4", "L4", "L4", "*", "N", "W", "L4", "U"]),
-      new DataRowParser(["opiekunka 3", "N", "W", "N", "Errr ", "D", "N", "W", "L4", "U"]),
+      ["opiekunka 1", "N", "Error", "N", " ", "D", "N", "NotValid", "L4", "U"],
+      ["opiekunka 2", "NotVald", "L4", "L4", "L4", "*", "N", "W", "L4", "U"],
```

Comment:
jeśli chodzi o ten test to ptanie na tarze

Reply:
Gwiazdka ma być traktowana jako śmieć. Ale ma być zamieniana na wolne, a nie tak jak było wcześniej wolne lub zmianę niepracującą. Wymaga poprawy testu.